docs/solutions/sql_server_always_on_availability_groups/configuration.tf

```diff
@@ -0,0 +1,60 @@
+# configuration.tf - General configuration of SQL Always On deployment
+# Set variables here to meet your needs within your particular environment
+# Check both this file and sql.tf!
+
+# image_id - OCID of the Windows image to use.
+# BE SURE TO UPDATE WITH MOST CURRENT IMAGE.
+variable "image_id" {
```

Comment:
So are we having the user populate three image OCIDs when they only need one?  Also, when the image is upgraded to the latest revision, they have to go back and update three OCIDs?  I think putting the image OCID in env-vars is good as it is a configurable option, but three images in the template itself introduces errors.

Reply:
That's a fair point. However, if you keep it as a single OCID, then the user needs to be aware that that OCID must change when running in a different region. Generally, we want these solutions as close to running out of the box as possible, and it makes it a bit easier from that perspective if it works in every region without modification. So I'm in favor of using the map approach with all 3 regions, but if you want to stick with just one OCID then please add to env-vars and make this clear. Also, regardless, please add a comment about what image this is.

Reply:
Cool...will add this to env-vars as suggested.

Reply:
Please add this for release (using either of the changes we've discussed) - I want users to have a good chance that this will succeed the first time they run it, instead of having to track down an invalid image ocid for their region.

Reply:
Decided to add this as map tied to the configuration.tf and region variables...Removed from env-vars.